langchain/src/util/async_caller.ts

```diff
@@ -80,7 +80,9 @@ export class AsyncCaller {
               if (
                 error.message.startsWith("Cancel") ||
                 error.message.startsWith("TimeoutError") ||
-                error.message.startsWith("AbortError")
+                error.message.startsWith("AbortError") ||
+                error.message.includes("Request was aborted") ||
```

Comment:
@rattrayalex is there any way to detect these errors other than a constructor check? This will fail in minified environments...

Reply:
I think you want `error.name === 'AbortError'` ([docs](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal))

Reply:
Depending on what you want, we also export classes like `APIUserAbortError` and `APIConnectionTimeoutError`; you could check the `name` there too if you don't wan't to do an `instanceof`

Reply:
`error.name` I think just returned `Error` but will double check.

Reply:
got it, that might be because of this line: https://github.com/hwchase17/langchainjs/pull/2389/files/a0596c79e38da6f527a0a4578ba66bb53cfab999#diff-4d0f479be12d9b217277a26941bcfe72cfb9bc6b9bb9209099d6dd3294248bfbR412

which as I noted you may want to do something different with…